queries/racket/highlights.scm

```diff
@@ -58,7 +62,7 @@
 
 (list
  .
- (symbol) @function)
+ (symbol) @function (#not-has-ancestor? @function quote))
```

Comment:
just an fyi the ancestor functions are expensive - I'd pick one to use it on, and then another not - the latter one will override the looser former one, you can structure it how you'd like